src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_custom.py

```diff
@@ -665,6 +665,7 @@ def test_k8s_install_kubectl_create_installation_dir(self, logger_mock, mock_url
         finally:
             shutil.rmtree(temp_dir)
 
+    @unittest.skip('Update api version')
```

Comment:
Why need to update `acs` and `cosmosdb` cases in [Compute] PR ?

Reply:
Disks are used everywhere.

Reply:
Sure.